.github/workflows/ci.yml

```
@@ -11,7 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-node@v1
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '12.22.0'
```

```suggestion
        node-version: '14.x'
```

Per https://github.com/eslint/eslint/pull/14592#issuecomment-883787306